pkg/operator/staticpod/controller/node/node_controller.go

```diff
@@ -67,6 +71,14 @@ func (c *NodeController) sync(ctx context.Context, syncCtx factory.SyncContext)
 		return err
 	}
 
+	if c.extraNodeSelector != nil {
```

Comment:
could you add a comment explaining that the OR operation is not supported on keys and include a link to the issue you mentioned in the other comment?